workflows/export/vx_export_bmm.go

```diff
@@ -0,0 +1,178 @@
+package export
+
+import (
+	"fmt"
+	"path"
+	"strconv"
+	"strings"
+
+	"github.com/bcc-code/bccm-flows/activities"
+	"github.com/bcc-code/bccm-flows/common"
+	"github.com/bcc-code/bccm-flows/utils/wfutils"
+	"github.com/bcc-code/bccm-flows/workflows"
+	"go.temporal.io/sdk/workflow"
+)
+
+// https://support.spotify.com/us/artists/article/audio-file-formats/
+var aacBitrates = []string{"128k", "256k"}
+
+// This is what seems to be used today
+var mp3Bitrates = []string{"256k"}
+
+func VXExportToBMM(ctx workflow.Context, params VXExportChildWorklowParams) (*VXExportResult, error) {
+	logger := workflow.GetLogger(ctx)
+	logger.Info("Starting ExportToPlayout")
+
+	options := workflows.GetDefaultActivityOptions()
+	ctx = workflow.WithActivityOptions(ctx, options)
+
+	normalizedFutures := map[string]workflow.Future{}
+
+	// Normalize audio
+
+	for lang, audio := range params.MergeResult.AudioFiles {
+		ctx = workflow.WithChildOptions(ctx, workflows.GetDefaultWorkflowOptions())
+		future := workflow.ExecuteChildWorkflow(ctx, workflows.NormalizeAudioLevelWorkflow, workflows.NormalizeAudioParams{
+			FilePath:   audio,
+			TargetLUFS: -14.0,
+		})
+		normalizedFutures[lang] = future
+	}
+
+	normalizedResults := map[string]workflows.NormalizeAudioResult{}
+	for lang, future := range normalizedFutures {
+		normalizedRes := workflows.NormalizeAudioResult{}
+		err := future.Get(ctx, &normalizedRes)
+		if err != nil {
+			return nil, fmt.Errorf("failed to normalize audio for language %s: %w", lang, err)
+		}
+
+		logger.Debug("Normalized audio for language %s: %v", lang, normalizedRes)
+		normalizedResults[lang] = normalizedRes
+		params.MergeResult.AudioFiles[lang] = normalizedRes.FilePath
+	}
+
+	outputFolder, err := wfutils.GetWorkflowOutputFolder(ctx)
```

Comment:
off topic, at some point all "folder"/"dir" should be renamed to be consistent. i prefer "dir"